examples/mock.rs

```diff
@@ -54,8 +56,11 @@ async fn main() {
     make_create(&client).await.unwrap();
     assert!(recording.query().await.contains("CREATE TABLE"));
 
+    let metadata =
+        clickhouse::StructMetadata::new::<SomeRow>(vec![Column::new("no".to_string(), UInt32)]);
+
     // How to test SELECT.
-    mock.add(test::handlers::provide(list.clone()));
+    mock.add(test::handlers::provide(&metadata, list.clone()));
```

Comment:
Oh =(

Can we disable checks for such handlers? These types of tests are used to check behavior of actors in the case of rare conditions (for instance, some limit is reached by the server), so providing metadata everywhere is meaningless.